examples/listen/cid/main.go

```diff
@@ -32,13 +26,9 @@ func main() {
 			fmt.Printf("Client's hint: %s \n", hint)
 			return []byte{0xAB, 0xC1, 0x23}, nil
 		},
-		PSKIdentityHint:      []byte("Pion DTLS Server"),
-		CipherSuites:         []dtls.CipherSuiteID{dtls.TLS_PSK_WITH_AES_128_CCM_8},
-		ExtendedMasterSecret: dtls.RequireExtendedMasterSecret,
-		// Create timeout context for accepted connection.
-		ConnectContextMaker: func() (context.Context, func()) {
```

Comment:
Is it easy/possible to do 30 second timeout still with new APIs?

Reply:
Yes, you would essentially call `conn.HandshakeContext(<30 second timeout context>)` manually before the first read/write. In other words, the sequence of calls would be something like:

```go
for {
  conn := lis.Accept()
  go func() {
    if err := conn.Handshake(<30 second timeout context>); err != nil {
      ...
    }
    // regular read/write stuff
  }()
}
```